phys2bids/interfaces/txt.py

```diff
@@ -9,48 +9,46 @@
 from phys2bids.physio_obj import BlueprintInput
 
 
-def populate_phys_input(filename, chtrig):
+def labchart_read(channel_list, chtrig, header=[]):
```

Comment:
```suggestion
def read_labchart(channel_list, chtrig, header=[]):
```

Reply:
You want me to change the function name?

Reply:
names changed